third-party/realdds/src/dds-notification-server.cpp

```diff
@@ -83,9 +83,13 @@ dds_notification_server::dds_notification_server( std::shared_ptr< dds_publisher
         }
     } );
 
-    dds_topic_writer::qos wqos( RELIABLE_RELIABILITY_QOS );
-    wqos.history().depth = 10;                                                                      // default is 1
-    wqos.endpoint().history_memory_policy = eprosima::fastrtps::rtps::DYNAMIC_RESERVE_MEMORY_MODE;  // TODO: why?
+    dds_topic_writer::qos wqos( RELIABLE_RELIABILITY_QOS ); //RELIABLE_RELIABILITY_QOS default but worth mentioning
+    // Writer can finish reader-writer handshake and start transmitting the discovery messages before the reader have
+    // finished the handshake and can receivethe messages.
+    // If history is too small writer will not be able to re-transmit needed samples.
+    // Setting history to cover known streams in Safety Camera plus some spare
```

Comment:
Not sure you want to mention SC... let's keep it generic, maybe `to cover intended use-cases`? :)